test/functional/collations_tests.js

```diff
@@ -633,7 +633,7 @@ describe('Collation', function() {
     }
   });
 
-  it('Fail due to no support for collation', {
+  it.skip('Fail due to no support for collation', {
```

Comment:
This test is now failing, because `decorateWithCollation` prevents the `collation` option from being passed to the find command if the `maxWireVersion` is not high enough. So in this test, the `find` command will disregard the given `collation` and not error. However, this is not compliant with the [spec](https://github.com/mongodb/specifications/blob/master/source/collation/collation.rst#require-maxwireversion-5) which states that the driver must error, and should be fixed later with [NODE-1576](https://jira.mongodb.org/browse/NODE-1576)

Reply:
Hm this is difficult because it seems like we're breaking compatibility either way. This is showing us that we used to return an error in this particular case and no longer do, but now are not ever generating an error. At very least I think we should return an error just for this method, noting in NODE-1576 that we should eventually return an error in all cases where its not supported